### PR TITLE
Project Detail Page Initial Design #408xt8

### PR DESCRIPTION
**Pull request by mahamzayed**
# Description

The purpose of this PR is to have the initial design for the project details page set up, along with responsive mobile design. Please note that the mobile design is based on the [mobile design](https://app.abstract.com/projects/e709ca90-5ba9-11e9-9eaa-59df656d954a/branches/bfc80129-c2a7-410f-aec3-67d6d2f2843b/collections/b237e501-06aa-4681-b76f-217f7df902c4?collectionLayerId=f7b3bf37-eabd-4ad3-9d0e-d054f58d8ebb&mode=build&selected=3796769721-15C46492-26C9-49E9-A29A-E7A85FC53F26) of the dataset details page and may need a little bit of tweaking upon confirmation with the design team.

All data is hardcoded as of now. Next steps is to hook up everything to contentful to get the data.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Navigate to the [Project Details](https://localhost:3000/project-details) page. Everything should look and behave according to the [designs](https://app.abstract.com/projects/e709ca90-5ba9-11e9-9eaa-59df656d954a/branches/bfc80129-c2a7-410f-aec3-67d6d2f2843b/collections/8b24655b-3779-4608-b258-1f9ee04b060f?collectionLayerId=de217bd7-8a64-423b-8867-054b63a9e262&mode=build&selected=1308687528-9F3C830A-DFD4-4049-A1E5-66661150F35F)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: components/DetailsTabs/DetailsTabs.vue [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'DetailsTabs',
+  data() {
+    return {
+      tabTypes: [
```
Comment (cameronbaney) [inline components/DetailsTabs/DetailsTabs.vue]: These will most likely need to be `props` so the parent component can pass down the tab data to this component.

**Comment by mahamzayed** [comment]
> We had talked about the `DetailsHeader` and `DetailsTabs` components using slots. Why did you decide against that? Unless this will be coming in the later stages of the feature branch?
> 
> I saw your note about the mobile design, so I won't give any feedback on the breaking tab design for now as I'm sure that's what you are talking about.
> 
> I noticed a few discrepancies with the design:
> 
> * Padding around the header and tabs boxes
> * Color for the active tab
> * Color for the `h3` above the project's title: "CARDIO-RESPIRATORY SYSTEM: HEART"

@cameronbaney Yes, the components will be using slots as we discussed. This will be coming in a later version of the page when I start hooking up contentful.

Thanks for addressing the design discrepancies. I'll address.

File: components/DetailsTabs/DetailsTabs.vue (hunk 1 above)
Comment (mahamzayed) [reply components/DetailsTabs/DetailsTabs.vue]: @cameronbaney I will address this when I change the component to pass in props in my next PR with contentful.

**Comment by mahamzayed** [comment]
@cameronbaney Addressed design discrepancies, with the exception of responsive mobile design and the props for the `DetailsHeader` component.

**Pull request merged**